### DISCUSSION TITLE
Autocomplete (CTRL+SPACE) doesn't Work with optional quoting backtick names

### DISCUSSION THREAD

**Discussion by estaleiroweb** [dbeaver/dbeaver#18536] [DBeaver Development]
When we are editing a view in MySQL/Mariadb that view is shown name with between backtick.
At to do autocomplete (CRTL+SPACE) with alias name without backtick this one doesn't work.

Ex: 
$\color{#f00}{It dosen't work:}$
~~~sql
CREATE VIEW `view_name` AS
SELECT 
a.<CTRL+SPC=listless>
FROM `database`.`table` `a`;
~~~

OR
~~~sql
CREATE VIEW `view_name` AS
SELECT 
`a`.<CTRL+SPC=listless>
FROM `database`.`table` a;
~~~

$\color{#009f00}{It works only like that:}$
~~~sql
CREATE VIEW `view_name` AS
SELECT 
a.<CTRL+SPC=listless>
FROM `database`.`table` a;
~~~

OR

~~~sql
CREATE VIEW `view_name` AS
SELECT 
`a`.<CTRL+SPC=listless>
FROM `database`.`table` `a`;
~~~

**Comment by LonwoLonwo** [comment]
Hello @estaleiroweb

Please tell me - what is the reason for using aliases in quoting backticks?
